orchestra/tests/helpers/workflow.py

```diff
@@ -552,6 +557,56 @@ def machine_task_function():
             },
         ],
     },
+    {
+        'slug': 'sanitybot',
+        'name': 'SanityBot workflow',
+        'certifications': [],
+        'versions': [
+            {
+                'slug': 'sanitybot_workflow',
+                'name': 'SanityBot workflow',
+                'description': 'A description of the SanityBot workflow',
+                'sanity_checks': {
+                    'sanity_check_function': {
+                        'path': ('orchestra.tests.helpers.workflow'
+                                 '.check_project')
+                    },
+                    'check_configurations': {
+                        "friendly_check": {
+                            "handlers": [{
+                                "type": "slack_project_channel",
+                                "message": "Friendly message",
+                                "steps": ["step1"]
+                            }],
+                            "repetition_seconds": 86400
+                        },
+                    }
```

Comment:
Can we use this as an example of the sanity_checks blob? What would it look like with multiple checks? What happens to "deleted" checks?